### DISCUSSION TITLE
Need information on how to configure an appclient log file

### DISCUSSION THREAD

**Discussion by starksm64** [eclipse-ee4j/glassfish#25140] [Q&A] [answered]
## Environment Details
* GlassFish Version (and build number): GF 8.0.0-JDK17-M5
* JDK version: 17.0.12+7
* OS: Fedora 31
* Database: db-derby-10.15.2.0-bin 

----------

## Problem Description
I am working on the configuration for appclient based vehicle tests in EE11 TCK and need to port the EE10 ts.jte command.testExecuteAppClient to a command line usable with the new appclient protocol and command line launcher. One thing I need for debugging is more logging information from the glassfish8/glassfish/bin/appclient process.

It is possible to configure this to produce an application.log file controlled by a logging.properties JUL or other logging framework settings?

**Comment by OndroMih** [comment, marked as answer]
If you can, just edit the appclient script and add a custom `logging.properties` file with `-Djava.util.logging.config.file` property in the last line, which invokes the Java process.

You can also set the `AS_DEBUG` env var to `true`, that will print some debug info std err that might help you understand what's going on.

**Reply by dmatej** [reply]
I tried to look into the code, seems also `export AS_DEBUG=true` might print something to STDERR output. But for standard logs I would do the same as Ondro advised.
